site/gatsby-site/cypress/integration/wordcounts.js

```diff
@@ -0,0 +1,47 @@
+describe('The Word Counts Page', () => {
+  it('successfully loads', () => {
+    cy.visit('/summaries/wordcounts');
+  });
+
+  it('word count table should exist', () => {
+    cy.visit('/summaries/wordcounts');
```

Comment:
These CSS-based selectors should detect whether the table as it currently exists is present or not, but if/when the display of the table changes this test will naturally fail. I think this test as a basic "did it load" test would be fine just checking for the header text of `Word Counts`. Your next test, which looks at the contents of the table, is then looking at the more specific question of the contents of the table.